packages/bpk-component-datatable/src/BpkDataTable-test.js

```diff
@@ -222,6 +223,99 @@ describe('BpkDataTable', () => {
     expect(cell.text()).toBe('Some guy');
   });
 
+  it('should sort rows using custom sort when it is passed', () => {
+    let complexRows = [
+      {
+        name: 'Jose',
+        description: 'Software Engineer',
+        seat: { office: 'London', desk: 10 },
+      },
+      {
+        name: 'Rolf',
+        description: 'Some guy',
+        seat: { office: 'Barcelona', desk: 12 },
+      },
+      {
+        name: 'John',
+        description: 'Some other guy',
+        seat: { office: 'Barcelona', desk: 15 },
+      },
+    ];
+    let sortByValue = 'seat';
+    let sortDirectionValue = 'DESC';
+    const sortFunction = ({ sortBy, sortDirection }) => {
+      if (sortBy === 'seat') {
```

Comment:
We could probably make this sortFunction a little short as you are never testing pressing the sort in any other way so this else would never get tested.

Unless we want to add another part of the test which is to click another column in the rendered output to really test this.